src/gluonnlp/layers.py

```diff
@@ -144,6 +146,40 @@ def __repr__(self):
                                            for k, v in self._kwargs.items()]))
 
 
+@use_np
+class RMSNorm(HybridBlock): 
+    """Apply root mean square layer normalization to n-dimensional input array, 
+    where we do not substract mean in the numerator. 
+
+    For more details, see the paper: https://arxiv.org/pdf/1910.07467.pdf
+    """
+    def __init__(self, in_channels, center=True, scale=True, 
```

Comment:
We may change center to `use_bias`. and then set it to `False` by default to align with T5.

Reply:
Resolve it because I noticed that we have been using `center` in LayerNorm.